fd-beta-version/src/v1/model/dbpayments.ts

```diff
@@ -0,0 +1,30 @@
+import { appdb } from "./appdb";
+
+export class dbPayments extends appdb {
+  constructor() {
+    super();
+    this.table = "payments";
+    this.uniqueField = "id";
+  }
+
+  async createPayment(
+    orderId: number,
+    paymentMethod: string,
+    totalAmount: number
+  ): Promise<any> {
+    const paymentData = {
+      order_id: orderId,
+      method: paymentMethod,
+      amount: totalAmount,
+      status: "processing",
+    };
+
+    try {
+      const result = await this.insertRecord(paymentData);
+      return result;
```

Comment:
If there is not any logic and simply insert or select operation then you can write directly it in controller file also

Reply:
changes done